compiler/luci-interpreter/src/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ endif()
 
 message(STATUS "LUCI INTERPRETER BEGIN")
 
+set(LUCI_INTERPRETER_BINARY "luci_interpreter${LUCI_INTERPRETER_SUFFIX}")
+set(LUCI_INTERPRETER_CORE "luci_interpreter_core${LUCI_INTERPRETER_SUFFIX}")
+set(LUCI_INTERPRETER_KERNELS "luci_interpreter_kernels${LUCI_INTERPRETER_SUFFIX}")
+set(LUCI_INTERPRETER_LOADER "luci_interpreter_loader${LUCI_INTERPRETER_SUFFIX}")
+
```

Comment:
You could add `_` after `luci_interpreter` for readability.

Reply:
It will break the build